### ISSUE TITLE
MX types failing CI 

### ISSUE THREAD

**Issue by vayuda** [pytorch/ao#356]
Mark said you should be familiar with this @vkuzo 
https://github.com/pytorch/ao/actions/runs/9487632431/job/26150568432?pr=347
```
 =================================== FAILURES ===================================
  ___________________ test_inference_compile_simple[fp4_e2m1] ____________________
  
  elem_dtype = 'fp4_e2m1'
  
      @pytest.mark.skipif(not torch.cuda.is_available(), reason="CUDA not available")
      @pytest.mark.parametrize("elem_dtype", SUPPORTED_ELEM_DTYPES)
      def test_inference_compile_simple(elem_dtype):
          """
          Smoke test for inference compile
          """
          if elem_dtype in (torch.float8_e4m3fn, torch.float8_e5m2):
              if not IS_CUDA_GE_89:
                  pytest.skip("CUDA capability >= 8.9 required for float8 in triton")
          m = nn.Sequential(nn.Linear(4, 6, bias=False, dtype=torch.bfloat16))
          m = m.cuda()
          m_mx = copy.deepcopy(m)
          block_size = 2
          swap_linear_with_mx_inference_linear(m_mx, elem_dtype, block_size)
          m_mx = torch.compile(m_mx, fullgraph="true")
      
          x = torch.randn(2, 4, device="cuda", dtype=torch.bfloat16)
          y_ref = m(x)
          y_mx = m_mx(x)
          sqnr = compute_error(y_ref, y_mx)
          if elem_dtype is torch.float8_e4m3fn:
              assert sqnr >= 20.0
          else:
  >           assert sqnr >= 14.0
  E           AssertionError: assert tensor(13.1250, device='cuda:0', dtype=torch.bfloat16, grad_fn=<MulBackward0>) >= 14.0
  
  test/prototype/mx_formats/test_mx_linear.py:192: AssertionError
```

**Comment by msaroufim** [comment]
So turns out this was fixed by just changing the sqnr to 13.5 - perhaps not a super satisfying answer but just unblocks for now https://github.com/pytorch/ao/pull/344

**Comment by jerryzh168** [comment]
yeah this seems to be a flaky test, I get 13.875 in the CI
